released/discovery_center/mission_4327/setup_subaccount_cf/README.md

```diff
@@ -0,0 +1,70 @@
+# Terraform Script to setup SAP BTP Account 
+The [Terraform provider](https://registry.terraform.io/providers/SAP/btp/latest) for SAP Business Technology Platform (BTP) enables you to automate the provisioning, management, and configuration of resources on SAP Business Technology Platform. By leveraging this provider, you can simplify and streamline the deployment and maintenance of BTP services and applications. Currently the BTP Terraform provider is available in beta for non productive usage. 
+
+The [Terraform](https://www.terraform.io/) script  documented here automates the setup of an SAP BTP subaccount based on a predefined template. The scripts can be used create BTP Sub Account with Cloud Foundry or Kyma runtime. The terraform scripts does the below configuration after creating a BTP Subaccount
+
+1. Configure the BTP entitlements required to complete the Discovery Center mission
+2. Enables BTP runtime CF
+3. Create the neccessary Subscription to Software as Service(SaaS) Apps (SAP Business Application Studio(BAS), SAP Build Work Zone, standard edition etc)
+4. Assgn users the neccessary roles required to access the SaaS Apps e.g BAS
+5. Add additional users to the subaccount.
+
+## Prerequisites
```

Comment:
We also offer a devcontainer in this repo. This should be mentioned here as it lowers the entry barrier of installing tools.

This also enables the usage of Codesapces so no local cloning of the code is necessary

Reply:
Same comments apply for the other README

Reply:
Wil be added to global README - resolve